data/json/items/generic/casing.json

```diff
@@ -44,6 +44,22 @@
     "volume": "3 ml",
     "weight": "1 g"
   },
+  {
+    "id": "22_casing_new_short",
+    "looks_like": "22_casing_new",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "price": 90,
+    "price_postapoc": 0,
+    "name": { "str": "unused .22 blank casing" },
+    "symbol": "=",
+    "color": "yellow",
+    "description": "An unfired, like-new .22 round casing, with the primer still intact. This one appears to be sourced from a blank round and is thus shorter.",
```

Comment:
```suggestion
    "description": "An unfired, like-new .22 short casing, with the priming compound still intact.",
```

If you're a little less specific about the casing this can also be the casing for .22 "CB" and a weird subsonic loading of .22 LR.
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/7764202/2e8f7d89-fe0c-47ad-96a0-10d853ff2733)